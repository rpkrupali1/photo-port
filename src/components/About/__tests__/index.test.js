import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import About from "..";
const { asFragment } = render(<About />);

afterEach(cleanup);

describe("About components", () => {
  //render about tests
  // verify that the component is rendering
  it("renders", () => {
    render(<About />);
  });

  //compare snapshot versions of the DOM node structure
  it("matches snapshot DOM node structure", () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
