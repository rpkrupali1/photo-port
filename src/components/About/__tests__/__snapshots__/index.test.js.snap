// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About components matches snapshot DOM node structure 1`] = `<DocumentFragment />`;
